sdk/test/AzureTest/AzureTestTests/AzureTestTests.swift

```diff
@@ -0,0 +1,66 @@
+// --------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// The MIT License (MIT)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the ""Software""), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+//
+// --------------------------------------------------------------------------
+
+import XCTest
+import Foundation
+import DVR
+import AzureTest
+
+class AzureTestTests: XCTestCase {
+
+    let fakeData = try! String(contentsOfFile: "/Users/jairmyree/Downloads/output-onlineyamltools.json")
+    
+    var fakeRequest : URLRequest?
+    
+    var fakeResponse : URLResponse?
+    
+    var fakeResponseData : Data?
+    
+    
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        let dataDictionary = try! JSONSerialization.jsonObject(with: fakeData.data(using: .utf8)!, options: []) as! [String:Any]
+        fakeRequest = URLRequest(url: URL(string: try! dataDictionary.arrayForKey("interactions").dictionaryForIndex(0).dicionaryForKey("request").stringForKey("uri"))!)
+        fakeRequest?.httpBody = try! dataDictionary.arrayForKey("interactions").dictionaryForIndex(0).dicionaryForKey("request").stringForKey("body").data(using: .utf8)
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testCleanRequestURL() throws {
```

Comment:
Take a look at tests in other modules to see how to name this. Something like:
```swift
func test_scrubbingRequest_removesSubscriptionId
```
Or something like that. Generally we don't use underscore in Swift, except for these test names.